src/modules/curations/itemCuration/reducer.ts

```diff
@@ -43,7 +60,47 @@ export function itemCurationReducer(state: ItemCurationState = INITIAL_STATE, ac
         },
         error: null
       }
+    }
+
+    case PUBLISH_THIRD_PARTY_ITEMS_SUCCESS: {
+      const { itemCurations } = action.payload
```

Comment:
we can add collectionId here, used below

Reply:
updated!